platforms/r3-corda/charts/mongodb-tls/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```

Comment:
description needs to be proper. also chart name should match folder name "mongodb-tls" otherwise helm lint will fail.